ui/page/playlists/internal/collectionsListMine/view.jsx

```diff
@@ -177,15 +177,15 @@ export default function CollectionsListMine(props: Props) {
 
   React.useEffect(() => {
     if (savedCollectionIds.length > 0) {
-      doFetchItemsInCollections({ collectionIds: savedCollectionIds });
+      doResolveClaimIds(savedCollectionIds);
     }
-  }, [doFetchItemsInCollections, savedCollectionIds]);
+  }, [doResolveClaimIds, savedCollectionIds]);
 
   function handleCreatePlaylist() {
     doOpenModal(MODALS.COLLECTION_CREATE);
   }
 
-  if (areBuiltinCollectionsEmpty && !hasCollections) {
+  if (areBuiltinCollectionsEmpty && !hasCollections && !isFetchingCollections) {
```

Comment:
:question: I think Queue is not meant to appear here, because `Publish` in the context menu doesn't make sense, and when `Edit` is selected, most of the GUI says it's not supposed to be editable.

<img width="697" alt="image" src="https://user-images.githubusercontent.com/64950861/189853728-bba4f411-d195-424e-bb51-239cb91650ba.png">

<img width="231" alt="image" src="https://user-images.githubusercontent.com/64950861/189853872-047f635d-292f-4734-8bc0-c1f31d61e641.png">